proposals/vendir/001-lazy-synching-on-stable-config/README.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Lazy Synching"
+authors: [ "Fritz Duchardt <fritz.duchardt.ext@gec.io>" ]
+status: "draft"
+approvers: [ ]
```

Comment:
Approvers should be:
[ @kumaritanushree, @vmunishwar,  @neil-hickey,  @joaopapereira ]